modules/calib3d/test/test_usac.cpp

```diff
@@ -480,4 +480,38 @@ TEST(usac_solvePnPRansac, regression_21105) {
     EXPECT_EQ(inliers, inliers_copy);
 }
 
+TEST(usac_Essential, maxiters) {
+    cv::RNG &rng = cv::theRNG();
+    cv::Mat mask;
+    cv::Mat K1 = cv::Mat(cv::Matx33d(1, 0, 0,
+                                     0, 1, 0,
+                                     0, 0, 1.));
+    const std::vector<int> flags = {USAC_DEFAULT, USAC_ACCURATE, USAC_PROSAC, USAC_FAST, USAC_MAGSAC};
+    const double conf = 0.99, thr = 0.5;
+    int roll_results_sum = 0;
+
+    for (int iters = 0; iters < 10; iters++) {
+        for (auto flag : flags) {
```

Comment:
It's make sense to implement it as parameterized test with flags as parameter. In case if failure the regression is more obvious as with loop. See example here: https://github.com/opencv/opencv/blob/4.x/modules/calib3d/test/test_affine2d_estimator.cpp#L46